controllers/pvc_storageclass_test.go

```diff
@@ -263,7 +275,9 @@ var _ = Describe("The default StorageClass is not StorageOS", func() {
 			err := k8sClient.Get(ctx, client.ObjectKey{Name: storageClass.Name}, &persistedSC)
 			Expect(err).NotTo(HaveOccurred(), "failed to fetch StorageClass")
 
-			By("Expecting the PVC to be patched")
+			By("Expecting the PVC has annotated with StorageClass")
```

Comment:
Again, just a nit, but this would read better: `Expecting the PVC has the StorageClass annotation`